commit ed95bf5638748068ef05ccbaa0a008b50ebecb42
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Sep 3 03:00:32 2017 +0200

    hurd: Fix p{read,write}v64v2.c build
    
            * sysdeps/posix/preadv64v2.c: Use off64_t instead of off_t.
            * sysdeps/posix/pwritev64v2.c: Use off64_t instead of off_t.

diff --git a/sysdeps/posix/preadv64v2.c b/sysdeps/posix/preadv64v2.c
index ab71f027d4..263f9b77c4 100644
--- a/sysdeps/posix/preadv64v2.c
+++ b/sysdeps/posix/preadv64v2.c
@@ -20,7 +20,7 @@
 #include <sys/uio.h>
 
 ssize_t
-preadv64v2 (int fd, const struct iovec *vector, int count, off_t offset,
+preadv64v2 (int fd, const struct iovec *vector, int count, off64_t offset,
 	    int flags)
 {
   if (flags != 0)
diff --git a/sysdeps/posix/pwritev64v2.c b/sysdeps/posix/pwritev64v2.c
index bf62d74190..f340c8a46d 100644
--- a/sysdeps/posix/pwritev64v2.c
+++ b/sysdeps/posix/pwritev64v2.c
@@ -21,7 +21,7 @@
 
 /* Since we define no flags for pwritev2 just route to pwritev.  */
 ssize_t
-pwritev64v2 (int fd, const struct iovec *vector, int count, off_t offset,
+pwritev64v2 (int fd, const struct iovec *vector, int count, off64_t offset,
 	     int flags)
 {
   if (flags != 0)
